cmd/habitat/community/ipfs.go

```diff
@@ -0,0 +1,64 @@
+package community
```

Comment:
i don't have strong feels about this since having interchangeable backends is very much far in the future, but it might be harder to switch out ipfs if we make it in the community package instead of more like a lib

Reply:
yeah this stuff is in a bit of a weird state, because when the community actually starts the IPFS node, its through a proc running the ipfs-driver. But when we need to get the initial configuration for it we still need to do weird stuff like this. Maybe procs can implement an optional initialization step that runs on a single node before they are started or something.